### ISSUE TITLE
Method of calculating FLOPs

### ISSUE THREAD

**Issue by huukim136** [Lyken17/pytorch-OpCounter#117]
Hi,
Thank you for the very helpful code.
I have a question a bout how you calculate the FLOPs.
Guessing from the code, you only took into account the operation of modules in the the model.
How about other operation of 2 variables such as softmax,... 
I think we also need to consider this such operation.

Thank you.

**Comment by Lyken17** [comment]
thop is based on pytorch forward hooks, which should support multiple args already.
